docs/content/preview/secure/enable-authentication/authentication-ysql.md

```diff
@@ -11,26 +11,7 @@ menu:
 type: docs
 ---
 
-<ul class="nav nav-tabs-alt nav-tabs-yb">
-  <li >
-    <a href="../ysql/" class="nav-link active">
-      <i class="icon-postgres" aria-hidden="true"></i>
-      YSQL
-    </a>
-  </li>
-  <li >
-    <a href="../ycql/" class="nav-link">
-      <i class="icon-cassandra" aria-hidden="true"></i>
-      YCQL
-    </a>
-  </li>
-  <li>
-    <a href="../yedis/" class="nav-link">
-      <i class="icon-redis" aria-hidden="true"></i>
-      YEDIS
-    </a>
-  </li>
-</ul>
+{{<api-tabs>}}
 
```

Comment:
With these pages renamed, some links still point to old pages
enable-authentication/ysql
including the Enable auth landing page